Formula/tigmint.rb

```diff
@@ -17,23 +17,27 @@ class Tigmint < Formula
   depends_on "minimap2"
   depends_on "numpy"
   depends_on "python@3.9"
+  depends_on "samtools"
 
   def install
     ENV.prepend_path "PATH", libexec/"bin"
     xy = Language::Python.major_minor_version "python3"
     ENV.prepend_path "PYTHONPATH", libexec/"lib/python#{xy}/site-packages"
     inreplace "bin/tigmint-cut", "/usr/bin/env python3", Formula["python@3.9"].bin/"python3.9"
-    inreplace "bin/tigmint-molecule", "/usr/bin/env python3", Formula["python@3.9"].bin/"python3.9"
-    inreplace "bin/long-to-linked", "/usr/bin/env python3", Formula["python@3.9"].bin/"python3.9"
+    inreplace "bin/tigmint_molecule.py", "/usr/bin/env python3", Formula["python@3.9"].bin/"python3.9"
```

Comment:
For executables installed in `#{bin}`, I prefer the naming scheme `tigmint-*` with no `.py` extension. Renaming executables can also break users' existing shell scripts that assume the old name. Was there a reason for renaming this executable?

Reply:
Yes, so the reason was that in making the `tigmint_molecule_paf.py` script, which is a faster way to infer molecule extents for `tigmint-long`, to avoid code duplication I imported some functions from `tigmint-molecule`. Unfortunately, though, that naming convention doesn't work within python scripts for imports...